Website/app/disclaimer/page.tsx

```diff
@@ -0,0 +1,15 @@
+export default async function Disclaimer() {
+  const response = await fetch(
+    "https://raw.githubusercontent.com/AvinashAbbigeri/Drifty/disclaimers/disclaimer.txt",
```

Comment:
I have checked the website deployment and it works nice 🙂. So, you can change the link to the correct one.